sdk/webpubsub/azure-webpubsub/azure/webpubsub/_azure_web_pub_sub_service_restapi.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+
+from azure.core import PipelineClient
+from azure.core.protocol import HttpResponse
+from msrest import Deserializer, Serializer
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Dict, Optional
+
+    from azure.core.protocol import HttpRequest
+
+from ._configuration import AzureWebPubSubServiceRESTAPIConfiguration
+from .operations import HealthApiOperations
+from .operations import WebPubSubOperations
+
+
+class AzureWebPubSubServiceRESTAPI(object):
+    """AzureWebPubSubServiceRESTAPI.
+
+    :ivar health_api: HealthApiOperations operations
+    :vartype health_api: azure.webpubsub.operations.HealthApiOperations
+    :ivar web_pub_sub: WebPubSubOperations operations
+    :vartype web_pub_sub: azure.webpubsub.operations.WebPubSubOperations
+    """
+
+    def __init__(
+        self,
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        base_url = 'None'
+        self._config = AzureWebPubSubServiceRESTAPIConfiguration(**kwargs)
+        self._client = PipelineClient(base_url=base_url, config=self._config, **kwargs)
+
+        client_models = {}  # type: Dict[str, Any]
+        self._serialize = Serializer(client_models)
+        self._serialize.client_side_validation = False
+        self._deserialize = Deserializer(client_models)
+
+        self.health_api = HealthApiOperations(
+            self._client, self._config, self._serialize, self._deserialize)
+        self.web_pub_sub = WebPubSubOperations(
+            self._client, self._config, self._serialize, self._deserialize)
+
+    def _send_request(self, http_request, **kwargs):
+        # type: (HttpRequest, Any) -> HttpResponse
+        """Runs the network request through the client's chained policies.
```

Comment:
Give summary that points to `protocol` layer to show that's where you find preparers